src/utilities/HYPRE_utilities.h

```diff
@@ -222,6 +222,9 @@ HYPRE_Int HYPRE_ClearError(HYPRE_Int hypre_error_code);
 /* Set behavior for printing errors: mode 0 = stderr, mode 1 = memory buffer */
 HYPRE_Int HYPRE_SetPrintErrorMode(HYPRE_Int mode);
 
+/* Free the memory buffer allocated for storing error messages when using mode 1 for printing errors */
```

Comment:
Can you move this after `PrintErrorMessages()` and change the comment to "Clear the error messages in hypre and free any related memory allocated"

Reply:
Done in [e8521a6](https://github.com/hypre-space/hypre/pull/1190/commits/e8521a61c99931c696f0430a77a02e09152e7048)